app/uploaders/photo_uploader.rb

```diff
@@ -0,0 +1,35 @@
+# encoding: utf-8
+
+class PhotoUploader < CarrierWave::Uploader::Base
+
+
+
+  include CarrierWave::MiniMagick
+
+
+  include Sprockets::Rails::Helper
+
+  storage :fog
+
+
+  def store_dir
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+
+  version :tiny do
+    process :resize_to_fill => [20, 20]
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)